modules/help_editor/jsx/help_editor_helper.js

```diff
@@ -0,0 +1,39 @@
+class EditContent extends React.Component {
```

Comment:
Thought: not standard to have React component and jQuery in one file under jsx. For the next iteration, would like to pull this React component out into its own file as an exported component in modules/help_editor/jsx and put help_editor_helper.js back in modules/help_editor/js with no jsx.